com.unity.perception/Documentation~/Tutorial/Phase3.md

```diff
@@ -77,7 +77,8 @@ In order to make sure our builds are compatible with Unity Simulation, we need t
 
 Here, you can specify a name for the run, the number of Iterations the Scenario will execute for, and the number of Instances (number of nodes the work will be distributed across) for the run. This window automatically picks the currently active Scene and Scenario to run in Unity Simulation.
 
-* **:green_circle: Action**: Name your run `FirstRun`, set the number of Iterations to `1000`, and Instances to `20`. 
+* **:green_circle: Action**: Name your run `FirstRun`, set the number of Iterations to `1000`, and Instances to `20`.
+* **:green_circle: Action**: Click `Randomize` to generate a seed for your runs.
```

Comment:
```suggestion
* **:green_circle: Action**: If you'd like to use a new random seed for this run of your Scenario, click `Randomize` to generate a new seed.
```

Reply:
Nice catch, updating execution ID and project ID to match screenshots